_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
+    * Go to your **Applications/Utilites** folder. Double-click **Terminal**.
+* Once the **Terminal** is open, you will need to navigate to the folder in which you put all the files. As an example, let's say you put them on the Desktop. You would type the following in your **Terminal** and hit "return":
+
+```    
+cd ~/Desktop
+```
+
+cd stands for change directory. Once you hit "return", you will be working inside the Desktop. For another example, let's say you put your files in a folder called KittyCADPractice within the Documents folder. This would look like so:
+
+```
+cd ~/Documents/KittyCADPractice
+```
+
+You will now be working inside the folder you created called KittyCADPractice.
+* Once inside the correct folder, input the following command into **Terminal**. (Note: if you are signed in and have made an API token, we auto-populated your token below)
+
+<Token>
+    {(token, platform) =>
+        [
+            platform === 'windows'
+                ? `$env:KITTYCAD_API_TOKEN = '${token}'`
+                : `export KITTYCAD_API_TOKEN="${token}"`,
+            'pip3 install virtualenv',
+            'python3 -m venv venv',
+            platform === 'windows'
+                ? 'venv/Scripts/Activate.ps1'
+                : 'source venv/bin/activate',
+            'pip3 install -r requirements.txt',
+            'python3 convert.py',
+        ].join('\n')
+    }
+</Token>
+
+As long as everything went smoothly, you will see the following output:
+
+```
+File conversion id: <your-conversion-id>
+File conversion status: Completed
+Saving output to ./output.stl
+```
+
+**Note:** depending on the version of Python you have installed, the "Python3" command might differ.
+
+Congrats! You just make your first successful API command using a Python script. We offer plenty of API endpoints for you to use (file conversions, calculate center of mass, convert units, and more). Check them out [here!](/docs/api)
+
+### Windows
+* Once you download the .zip file and place it somewhere you can easily access, extract the contents of your zip file. You can do this by right-clicking on the .zip file and selecting **Extract Here**. This will unpack the zip file and place all the contents into the current directory it's in. 
+* You will need to install Python. Install the latest Python specific for your machine here: [Install Python](https://www.python.org/downloads/)
+    * **make sure you download the correct version for Windows**
+* Once you download Python, you will need to open your **Command Prompt**. One way to open the **Command Prompt** is to click on the Start button and type "cmd" and press enter
```

Comment:
Oh I think we should be recommending powershell. command prompt is kinda legacy at this point. Plus this bit
```
<Token>
    {(token, platform) =>
        [
            platform === 'windows'
                ? `$env:KITTYCAD_API_TOKEN = '${token}'`
                : `export KITTYCAD_API_TOKEN="${token}"`,
            'pip3 install virtualenv',
            'python3 -m venv venv',
            platform === 'windows'
                ? 'venv/Scripts/Activate.ps1'
                : 'source venv/bin/activate',
            'pip3 install -r requirements.txt',
            'python3 convert.py',
        ].join('\n')
    }
</Token>
``` 
I had setup with powershell in mind. I would be surprised if it worked in command prompt.

Reply:
Didn't even know there was a difference